quiche/src/path.rs

```diff
@@ -843,6 +843,19 @@ impl PathMap {
 
         Ok(())
     }
+
+    /// Configures path MTU discovery on all existing paths.
+    pub fn set_discover_pmtu_on_existing_paths(
+        &mut self, discover: bool, max_send_udp_payload_size: usize,
+    ) {
+        for (_, path) in self.paths.iter_mut() {
+            path.pmtud.enable(discover);
```

Comment:
What calls path.pmtud.enable on new path creation?

I think we're not modifying `Config {pmtud: bool}` in `lib.rs`.  See call to `path::PathMap::new`

https://github.com/cloudflare/quiche/blob/5a463e767b3c8999a5ce80097879f809ad973957/quiche/src/lib.rs#L1975

Reply:
> What calls path.pmtud.enable on new path creation?
> 
> I think we're not modifying `Config {pmtud: bool}` in `lib.rs`. See call to `path::PathMap::new`
> 
> https://github.com/cloudflare/quiche/blob/5a463e767b3c8999a5ce80097879f809ad973957/quiche/src/lib.rs#L1975

We can't modify `Config {pmtud: bool}` in the middle of the handshake, and I think only one `PathMap` is constructed per connection even if you're intending to utilize multiple `Path`s (they just get inserted into that `PathMap`. Also `config.pmtud` is only used for enabling PMTUD on the very initial path. Right now we don't support connection migration, so the initial path is the only one we use, but in the future we'll need to ensure that we enable pmtud for all `Path`s that need it. Out of scope for this PR though imo